core/src/Kokkos_View.hpp

```diff
@@ -165,7 +165,7 @@ class View : public Impl::BasicViewFromTraits<DataType, Properties...>::type {
   using execution_space      = typename traits::execution_space;
   using memory_space         = typename traits::memory_space;
   using memory_traits        = typename traits::memory_traits;
-  using host_mirror_space    = typename traits::host_mirror_space;
```

Comment:
This is a public type or? Can't just remove it, need to deprecate it.

Reply:
Fixed.